### PR TITLE
docs: Updated PayBC Sequence and State Diagrams

### PR DESCRIPTION
**Pull request by john-fletcher-aot**
# Description

Update PayBC sequence diagram to reflect actual behaviour. Added state diagram to identify possible states after failure in any part of the chain.

Fixes # ORV2-1317 (partial

## Type of change

- [X] Documentation

# How Has This Been Tested?

N/A

## Checklist

- [X] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (N/A)
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have already been accepted and merged

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://onroutebc-1149-frontend.apps.silver.devops.gov.bc.ca)
- [Vehicles](https://onroutebc-1149-vehicles.apps.silver.devops.gov.bc.ca/api)
- [Dops](https://onroutebc-1149-dops.apps.silver.devops.gov.bc.ca/api)
- [TPS-Migration](https://onroutebc-1149-tps-migration.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/onroutebc/actions/workflows/analysis.yml)

After merge, new images are promoted to:
- [Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge.yml)

### CODE REVIEW

File: common/payment/PayBC Sequence.md [hunk 1]
```diff
@@ -2,32 +2,33 @@
...
-    participant PayBC
-    participant CFS
     Browser->>onRouteBC: Request to pay for permit
-    onRouteBC->>PayBC: Generate token
-    note over onRouteBC,PayBC: Supply client id and secret
-    PayBC-->>onRouteBC: Send bearer token
     onRouteBC->>onRouteBC: Generate transaction number
     onRouteBC->>onRouteBC: Save transaction number to DB
-    onRouteBC->>onRouteBC: Set permit status to 'WAITING FOR PAYMENT'
+    onRouteBC->>onRouteBC: Set permit status to 'WAITING_PAYMENT'
```
Comment (erikataot) [inline common/payment/PayBC Sequence.md:8]: nice 👍

File: common/payment/PayBC Sequence.md [hunk 2]
```diff
@@ -2,32 +2,33 @@
...
     onRouteBC-->>Browser: Send redirect to PayBC
-    Browser->>PayBC: Forward to payment page
-    note over Browser,PayBC: Include all transaction details such as<br/>transaction number, amount, and auth token
-    PayBC-->>Browser: Display payment details form
-    Browser->>PayBC: Supply payment information (credit card)
-    PayBC->>PayBC: Process payment
-    note over PayBC: Assume for this diagram<br/>that the payment is<br/>successful
-    PayBC-->>Browser: Send redirect with details of transaction
+    Browser->>PayBC: Forward to PayBC
+    note over Browser,PayBC: Include all transaction details such as<br/>transaction number, amount, hash, etc.
```
Comment (erikataot) [inline common/payment/PayBC Sequence.md:11]: is the `etc` important to itemize?

**Review by praju-aot — APPROVED** [review]
Approved

File: common/payment/PayBC Sequence.md (hunk 2 above)
Comment (john-fletcher-aot) [reply common/payment/PayBC Sequence.md:11]: The transaction details sent over to PayBC are covered in the Postman collection that's been updated alongside this. Good note.

**Pull request merged**